.github/workflows/dco-check.yml

```diff
@@ -0,0 +1,17 @@
+name: DCO Check
+on: pull_request
+
+jobs:
+  dco:
+    runs-on: ubuntu-latest
+    name: DCO Check
+    steps:
+    - name: Get PR Commits
+      id: 'get-pr-commits'
+      uses: tim-actions/get-pr-commits@master
+      with:
+        token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
If (like me) you're nervous about giving away a token to a random GHA, here's where it's used:

https://github.com/tim-actions/get-pr-commits/blob/master/index.js#L16

Seems legit/necessary to me.

Reply:
It's fine though, the token on pull_request is read only, it can comment on PRs I think